docs/content/ActionMenu.mdx

```diff
@@ -0,0 +1,32 @@
+---
+title: ActionMenu
+---
+
+An `ActionMenu` is an ActionList-based component for creating a menu of actions that expands through a trigger button.
+
+## Default example
+
+```jsx live
+<ActionMenu
+  anchorContent="Menu"
+  onAction={({text}) => console.log(text)}
+  items={[
+    {text: 'New file'},
+    ActionMenu.Divider,
+    {text: 'Copy link'},
+    {text: 'Edit file'},
+    {text: 'Delete file', variant: 'danger'}
+  ]}
+/>
+```
+
```

Comment:
Could you add an example that uses `groupMetadata`?

Reply:
done, just pulled from our `ActionList` docs on this one and kept them parallel.